pkg/reconciler/kubernetes/tektoninstallerset/tektoninstallerset.go

```diff
@@ -177,5 +177,11 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, installerSet *v1alpha1.T
 	// Mark all deployments ready
 	installerSet.Status.MarkAllDeploymentsReady()
 
+	// job
+	err = installer.IsJobCompleted()
+	if err != nil {
+		return err
+	}
+
```

Comment:
this should be something handled in a reconciler hub reconciler.

At present let us use tektoninstallerset to create resources and check if they are created properly. ie, clusterOperations related things.

 Checking for completion of a job seems to be something lifcycle related. Let the installerset create it, then the lifecycle aware tektonhub reconciler can check the job completion status and proceed.